sig-robotics/README.md

```diff
@@ -7,11 +7,29 @@ The [charter](charter.md) defines the scope and governance of the Robotics Speci
 
 ## Meetings
 
-
-* Regular SIG Meeting: [Tuesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (weekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
- * [Meeting notes and Agenda]().
- * [Meeting Calendar]() | [Subscribe]()
+* Regular SIG Meeting: [Tuesday at 10:00 UTC+8](https://zoom.us/my/kubeedge) (biweekly, starts on 22th, June 2021). [Convert to your timezone](https://www.thetimezoneconverter.com/?t=10%3A00%20am&tz=GMT%2B8&).
+ * [Meeting notes and Agenda](https://docs.google.com/document/d/1i-DTZTsj2GjKHlocpkyzfEO--zYvx7sZ4bEFoJCy4YM/edit#).
+ * [Meeting Calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=MGRnMGJzdjhmY3E2aDhhOTZlOXV0M211dXBfMjAyMzAyMDdUMDIwMDAwWiBodWFuZzY4OTRAbQ)
 
 ## Contact
-- Slack: [#sig-robotics]()
+
+- [#sig-Robotics slack channel](https://kubeedge.slack.com)
+- [Open Community Issues](https://github.com/kubeedge/community/issues)
 - [Mailing list](https://groups.google.com/forum/#!forum/kubeedge)
+
+## Leadership
+
+### Chairs
+
+The Chairs of the SIG run operations and processes governing the SIG.
+
+*TODO*
+
+### Technical Leads
+
+The Technical Leads of the SIG establish new subprojects, decommission existing subprojects, and resolve cross-subproject technical issues and decisions.
+
+- Zhengguang Ou (@ugvddm), Huawei
+- Fisher Xu (@fisherxu), Huawei
```

Comment:
Can remove me at here :) But I'll still help the development of the SIG.  @ugvddm @JoeyHwong-gk

Reply:
Others looks good.

Reply:
thanks for comments, fixed